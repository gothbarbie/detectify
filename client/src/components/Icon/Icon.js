import React from 'react'
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
import PropTypes from 'prop-types'

const Icon = ({ icon }) => {
  return <FontAwesomeIcon icon={icon} />
}

Icon.propTypes = {
  icon: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
}

export default Icon
