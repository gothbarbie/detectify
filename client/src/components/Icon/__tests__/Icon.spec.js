import React from 'react'
import { shallow } from 'enzyme'

import Icon from '../Icon'
describe('', () => {
  it('renders', () => {
    const wrapper = shallow(<Icon />)

    expect(wrapper).toMatchSnapshot()
  })
})
