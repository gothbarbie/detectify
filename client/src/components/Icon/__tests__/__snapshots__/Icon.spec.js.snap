// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<FontAwesomeIcon
  border={false}
  className=""
  fixedWidth={false}
  flip={null}
  icon={null}
  inverse={false}
  listItem={false}
  mask={null}
  pull={null}
  pulse={false}
  rotation={null}
  size={null}
  spin={false}
  symbol={false}
  title=""
  transform={null}
/>
`;
